From 5d16fbb11a86b1a918a3e4a0620ad229747c0e18 Mon Sep 17 00:00:00 2001
From: Alexey Chernov <4ernov@gmail.com>
Date: Wed, 20 May 2020 21:04:19 +0300
Subject: [PATCH] Include `neon2sse` third party header library

It is optionally used in quantized operators if certain SIMD
set is available, so we need to include this path for that case.
---
 caffe2/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/caffe2/CMakeLists.txt b/caffe2/CMakeLists.txt
index 8025a7de3c..e042ae5139 100644
--- a/caffe2/CMakeLists.txt
+++ b/caffe2/CMakeLists.txt
@@ -795,6 +795,8 @@ ENDIF()
   target_include_directories(torch_cpu PRIVATE
     ${TORCH_ROOT}/third_party/miniz-2.0.8)
 
+  target_include_directories(torch_cpu PRIVATE
+    ${TORCH_ROOT}/third_party/neon2sse)
 
   install(DIRECTORY "${TORCH_SRC_DIR}/csrc"
     DESTINATION ${TORCH_INSTALL_INCLUDE_DIR}/torch
-- 
2.26.2

